examples/index.tsx

```diff
@@ -15,6 +15,11 @@ const App = () => {
             <Input name="a" horizontal label="horizontal input"/>
             <Input name="b" label="simple input"/>
             <Select name="c" label="horizontal select" options={[{value: 1, label: "form"}]} horizontal/>
+            <div className="div">
+                <Radio name="male" value="male"/>
```

Comment:
This is not a proper usage of Radio Group. It should look like this `<Radio name="male" values={["male", "female"]}/>`.
Also `gender` not `male` :)